test/runtests.jl

```diff
@@ -61,6 +61,11 @@ include(joinpath(examples_path, "kinsol_mkinTest.jl"))
 @test abs(minimum(residual)) < 1e-5
 end
 
+println("== handle tests")
+let
+@testset "Handle Tests" begin include("handle_tests.jl") end
```

Comment:
Instead of printing the file, if you just put it in a testset you get the same information. All of the Sundials tests should be testsets instead of let blocks. I only did the let block thing to match the current style, but they should really match the more modern style of diffeq:

https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/test/runtests.jl

The DataTables.jl example is complicated in unnecessary ways. I would like to avoid that style.